e2e/kube/cpu_mem_test.go

```diff
@@ -0,0 +1,46 @@
+package kube_test
+
+import (
+	"path"
+
+	cmdHelper "code.cloudfoundry.org/quarks-utils/testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("BOSHDeployment", func() {
+	When("specifying cpu/memory resources requests and limits in bpm configs", func() {
```

Comment:
```suggestion
	When("specifying cpu/memory resources requests and limits in the quarks.bpm config", func() {
```

Something like that, I'd try to make it clear that this our config, not a BOSH one. There might be a test in the future, that checks if we convert existing "BOSH bpm limit section" information correctly.

Reply:
Right! sounds fair :+1: